packages/lab/src/pagination/PageButton.tsx

```diff
@@ -35,6 +35,7 @@ export const PageButton = ({
       aria-current={isSelected ? "page" : undefined}
       variant="secondary"
       className={clsx(withBaseName("pageButton"), {
+        [withBaseName("pageButtonReadonly")]: disabled,
```

Comment:
This is confusing me, the class is called readonly but the application is done on disabled, and if it is disabled styling, why wouldn't we use the disabled selector?